src/logi_transform.erl

```diff
@@ -111,7 +111,7 @@ transform_call(logi, Severity0, {_, _, _, Args} = Call, Loc = #location{line_or_
                 [Logger, {string, _, _} = Fmt] ->
                     Opts = {cons, LineOrAnno, {tuple, LineOrAnno, [{atom, LineOrAnno, logger}, Logger]}, {nil, LineOrAnno}},
                     logi_call_expr(Severity, Fmt, {nil, LineOrAnno}, Opts, Loc);
-                [Logger, {string, _, _} = Fmt, {nil, LineOrAnno} = Data] ->
+                [Logger, {string, _, _} = Fmt, {nil, _} = Data] ->
```

Comment:
When anno contains column, LineOrAnno that is bound at `#location{line_or_anno = LineOrAnno}` will not match here.